packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.facade.ts__tmpl__

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+import { Store } from '@ngrx/store';
+import { Actions } from '@ngrx/effects';
+
+import { <%= className %>State } from './<%= fileName %>.interfaces';
+import { <%= propertyName %>Query } from './<%= fileName %>.selectors';
+import { Load<%= className %> } from './<%= fileName %>.actions';
+
+@Injectable({
+  providedIn : 'root'
+})
+export class <%= className %>Facade {
```

Comment:
NgRx doesn't have a notion of the facade, so I don't think we should generate it by default. What we generate by default should be as closer to the default ngrx stuff as possible.

Could you remove everything facade related from this PR? Let's talk about it more and maybe add it separately as an opt-in thing.